apps/spruce/src/components/WaterfallModal/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from "react";
+import { useMutation } from "@apollo/client";
+import styled from "@emotion/styled";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+import MarketingModal from "@leafygreen-ui/marketing-modal";
+import Cookies from "js-cookie";
+import { useNavigate } from "react-router-dom";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { useToastContext } from "@evg-ui/lib/context/toast";
+import { useWaterfallAnalytics } from "analytics";
+import { SEEN_WATERFALL_BETA_MODAL } from "constants/cookies";
+import { getWaterfallRoute } from "constants/routes";
+import {
+  UpdateUserBetaFeaturesMutation,
+  UpdateUserBetaFeaturesMutationVariables,
+} from "gql/generated/types";
+import { UPDATE_USER_BETA_FEATURES } from "gql/mutations";
+import { useUserBetaFeatures } from "hooks/useBetaFeatures";
+import screenshot from "./screenshot.png";
+
+export const WaterfallModal: React.FC<{ projectIdentifier: string }> = ({
+  projectIdentifier,
+}) => {
+  const { sendEvent } = useWaterfallAnalytics();
+  const navigate = useNavigate();
+  const { userBetaSettings } = useUserBetaFeatures();
+  const dispatchToast = useToastContext();
+
+  const handleClose = (enabledBeta: boolean) => () => {
+    Cookies.set(SEEN_WATERFALL_BETA_MODAL, "true", { expires: 365 });
+    sendEvent({
+      name: "Viewed waterfall beta modal",
+      "beta.enabled": enabledBeta,
+    });
+    setOpen(false);
+  };
+
+  const [updateBetaFeatures] = useMutation<
+    UpdateUserBetaFeaturesMutation,
+    UpdateUserBetaFeaturesMutationVariables
+  >(UPDATE_USER_BETA_FEATURES, {
+    onCompleted: () => {
+      handleClose(true)();
+      navigate(getWaterfallRoute(projectIdentifier));
+    },
+    onError: () => {
+      handleClose(false)();
+      dispatchToast.error(
+        `Failed to enable waterfall beta. Visit your UI Settings page to update.`,
```

Comment:
nit. can use normal quotes
```suggestion
        "Failed to enable waterfall beta. Visit your UI Settings page to update.",
```